vaibhav/hangman/hangman.rb

```diff
@@ -0,0 +1,39 @@
+#hangman game
+
+def hangman
+    attempts = 6
+    words = %w[vaibhav]
```

Comment:
we can use .freeze here and if it is constant we can declare the constant variables

Reply:
this is for testing purpose